src/algebra/big_operators/basic.lean

```diff
@@ -1208,6 +1208,15 @@ lemma prod_pow_boole [decidable_eq α] (s : finset α) (f : α → β) (a : α)
   (∏ x in s, (f x)^(ite (a = x) 1 0)) = ite (a ∈ s) (f a) 1 :=
 by simp
 
+lemma prod_dvd {S : finset α} (g1 g2 : α → β) (h : ∀ a ∈ S, g1 a ∣ g2 a) : S.prod g1 ∣ S.prod g2 :=
+begin
+  classical,
+  apply @finset.induction_on' _ (λ S, S.prod g1 ∣ S.prod g2), { simp },
```

Comment:
Does `apply finset.induction_on' S` not work? That's surprising.

Reply:
Oh, it does, I'm not sure what I was doing wrong before.  Maybe I just tried `apply finset.induction_on'`